app/api/transportations_api.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+class TransportationsApi < Grape::API
+  format :json
+
+  helpers AuthorizationHelper, CarbonFootprintHelper
+
+  helpers AuthorizationHelper
+  before do
+    token = headers.fetch('auth_token', nil)
+    authorize_user(token) if token
+  end
+
+  namespace 'transportations' do
+    # POST /transportations
+    desc 'Create transportations' do
+      tags %w[transportations]
+      http_codes [
+        { code: 201, message: 'Create transportations' },
+        { code: 400, message: 'Bad request!' }
+      ]
+    end
+    desc 'Headers', {
+      headers: {
+        'auth_token' => {
+          description: 'Validates your identity',
+          optional: true
+        }
+      }
+    }
+
+    params do
+      optional :flights, type: Array[JSON] do
+        requires :from, type: String, desc: 'from', documentation: { param_type: 'body' }, default: 'Suceava'
+        requires :to, type: String, desc: 'to', documentation: { param_type: 'body' }, default: 'Kabul'
+      end
+      optional :public_transports, type: Array[JSON] do
+        requires :transport_type, type: Integer, desc: 'transport_type', documentation: { param_type: 'body' },
+                                  values: [0, 1], default: 0
```

Comment:
These values correspond to those used by the [model](https://github.com/AmprenTA/backend/pull/3/files#diff-b0c63653124cd419697246fb7128e8cdea545436ca38f3d686b1f1df45f26a46R4), we might consider using `ModelName.enum_name.values ` in order to avoid the lack of consistency.